### PR TITLE
T1180

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

File: src/core/popcorn-wrapper.js [hunk 1]
```diff
@@ -87,6 +87,7 @@ define( [ "core/logger", "core/eventmanager" ], function( Logger, EventManager )
 
           if( trackEvent.view ){
             var popcornEvent = _popcorn.getTrackEvent( _butterEventMap[ butterId ] );
+            trackEvent.popcornTrackEvent = popcornEvent;
```
Comment (secretrobotron) [inline src/core/popcorn-wrapper.js]: Do you need to do this inside of this `if` block? I don't think a view should be required to make this work. That if statement looks fishy anyway.

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @ScottDowne received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/92295e9f336aa61/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @ScottDowne received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/fe38d625ae2f988/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/92295e9f336aa61/output.txt

Total script time: 0.44 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/fe38d625ae2f988/output.txt

Total script time: 0.80 mins

Server is running on http://pr457.butyr.org/templates/test.html

**Pull request merged**